homeassistant/helpers/config_validation.py

```diff
@@ -552,6 +552,19 @@ def uuid4_hex(value):
     return result.hex
 
 
+def zwave_network_key(value):
```

Comment:
Since this is specific to the zwave integration, it should be put in the zwave integration component package somewhere.